controllers/providers/assistant/gslb.go

```diff
@@ -101,14 +101,24 @@ func (r *Gslb) CoreDNSExposedIPs() ([]string, error) {
 		err := coreerrors.New(errMessage)
 		return nil, err
 	}
-	IPs, err := utils.Dig(lbHostname, r.edgeDNSServers...)
-	if err != nil {
-		log.Warn().Err(err).
-			Str("loadBalancerHostname", lbHostname).
-			Msg("Can't dig CoreDNS service LoadBalancer FQDN")
-		return nil, err
+	return extractIPFromLB(lb, r.edgeDNSServers)
```

Comment:
Just trying to avoid [else statement](https://www.youtube.com/watch?v=gKaGpNnR07c&ab_channel=FriendlyNeighborhoodCoder) for little better readibility.  
```go
if len(coreDNSService.Status.LoadBalancer.Ingress) == 0 {
		errMessage := "no LoadBalancer ExternalIPs are found"
		log.Warn().
			Str("serviceName", coreDNSService.Name).
			Msg(errMessage)
		return nil, coreerrors.New(errMessage)
}
lb = coreDNSService.Status.LoadBalancer.Ingress[0]
return extractIPFromLB(lb, r.edgeDNSServers)
```

Reply:
Nice, updated